src/javascript/app_2/App/Components/Routes/route_with_sub_routes.jsx

```diff
@@ -32,7 +33,8 @@ const RouteWithSubRoutes = route => {
             );
         }
 
-        document.title = `${localize(route.title)} | Binary.com`;
+        const title = route.title ? `${ localize(route.title) } | ` : '';
+        document.title = `${ title }${default_title}`;
```

Comment:
we should either have space or no space after `{` for both variables to be consistent